futurenhs.app/helpers/util/form/index.ts

```diff
@@ -121,4 +121,10 @@ export const checkMatchingFormType = (formData: Record<any, any>, formId: string
 
     return Boolean(formData.body?.['_form-id'] === formId)
 
+}
+
+export const getFormField = (form: FormConfig, step = 0, name: string) => {
```

Comment:
What if I don't know what step a field is on? Ideally this should loop through the steps until it finds the matching field and then stop

Reply:
Now implemented as suggested and doesn't assume we know the step